DB/projects.json

```
@@ -3555,5 +3555,22 @@
         "tech": ["html", "css"]
       }
     ]
+  },
+   {
+    "github_username": "utsavbhattarai007",
+    "Projects": [
+      {
+        "github_url": "https://github.com/utsavbhattarai007/devhub-with-appwrite",
+        "title": "Hackathon project",
+        "description": "This project is the alternative twitter for developers",
+        "tech": ["reactjs", "appwrite"]
+      },
+      {
+        "github_url": "https://github.com/utsavbhattarai007/Portfolio",
+        "title": "My portfolio",
```

change the title to `Utsav Bhattarai's Portfolio`

```suggestion
        "title": "Utsav Bhattarai's Portfolio",
```